### PR TITLE
Allow init in empty repo

### LINKED ISSUE #8: Allow `init` in empty repo

### PR DESCRIPTION
**Pull request by atugushev**
Fixes #8.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -82,6 +82,11 @@ def tmp_cwd(tmp_path: pathlib.Path) -> typing.Iterator[pathlib.Path]:
     os.chdir(orig_cwd)
 
 
+@pytest.fixture()
+def empty_repo(tmp_cwd: pathlib.Path) -> pygit2.Repository:
```
Comment (lig) [inline tests/conftest.py:86]: Maybe make `root_repo` use this one.

File: tests/test_core.py [hunk 2]
```diff
@@ -33,6 +33,16 @@ def test_002_init(data_dir: pathlib.Path, root_repo: pygit2.Repository) -> None:
     )
 
 
+def test_002_init_with_empty_repo(
+    data_dir: pathlib.Path, empty_repo: pygit2.Repository
+) -> None:
+    core.init()
+
+    assert config.Config.load(path=pathlib.Path('wok.yml')) == config.Config.load(
+        path=data_dir / '002_wok.yml'
```
Comment (lig) [inline tests/test_core.py]: Let's call it `004` for the sample config and the test itself. I'd like to keep those completely isolated between tests.

**Review by lig — CHANGES_REQUESTED** [review]
Hey @atugushev! Awesome job. It's turned out to be that simple:)

Thanks for your contribution! Could you please address some comments from the review?

File: tests/conftest.py (hunk 1 above)
Comment (atugushev) [reply tests/conftest.py:86]: Had the same thoughts. Addressed in c465f32.

File: tests/test_core.py (hunk 2 above)
Comment (atugushev) [reply tests/test_core.py]: Addressed in c465f32.

**Comment by atugushev** [comment]
I've noticed that `wok start` doesn't work with empty repo either. Here is the tracking issue #12, better to be fixed in a followup this PR.

File: tests/test_core.py [hunk 3]
```diff
@@ -58,6 +48,16 @@ def test_003_init_in_branch(
     )
 
 
+def test_004_init_with_empty_repo(
+    data_dir: pathlib.Path, empty_repo: pygit2.Repository
+) -> None:
+    core.init()
+
+    assert config.Config.load(path=pathlib.Path('wok.yml')) == config.Config.load(
+        path=data_dir / '002_wok.yml'
```
Comment (lig) [inline tests/test_core.py]: `004_wok.yml`?

**Review by lig — CHANGES_REQUESTED** [review]

File: tests/test_core.py (hunk 3 above)
Comment (lig) [reply tests/test_core.py]: The idea is to copy the sample file and keep it as it is for this test only.

Comment (atugushev) [reply tests/test_core.py]: Sure! Addressed in 38cab18000ba58edae6b4a8e4e0554ee9afcb363.

**Pull request merged**